dotCMS/src/main/java/com/dotmarketing/portlets/structure/action/EditStructureAction.java

```diff
@@ -179,14 +179,15 @@ private Structure _loadStructure(ActionForm form, ActionRequest req, ActionRespo
 		ContentType type;
 		Structure struc = new Structure();
 		try{
-			type= APILocator.getContentTypeAPI(user).find(inodeString);
-			struc = new StructureTransformer(type).asStructure();
-		}
-		catch(NotFoundInDbException nodb){
+			if(!UtilMethods.isSet(inodeString)) {
+				type= ContentTypeBuilder.instanceOf(SimpleContentType.class);
+			} else {
+				type = APILocator.getContentTypeAPI(user).find(inodeString);
+				struc = new StructureTransformer(type).asStructure();
+			}
+		} catch(NotFoundInDbException nodb){

```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Either log or rethrow this exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1166)